ros2doctor/package.xml

```diff
@@ -9,7 +9,7 @@
 
   <depend>ros2cli</depend>
 
-  <exec_depend>python3-numpy</exec_depend>
+  <exec_depend>ifcfg-pip</exec_depend>
```

Comment:
Is there a Debian release of this package? Packages released into a ROS distro (like the packages in this repo) can't use packages from `PyPi` because `.deb`s can only depend on other Debian packages.

If there isn't a Debian release, an alternative is to release our own `deb`, which means creating a vendor package for `ifcfg`.

Reply:
vendor package for `ifcfg` at https://github.com/ros2/ifcfg_vendor

Reply:
need to change this dependency to `ifcfg_vendor`.

Reply:
do I also need to change to ifcfg_vendor on rosdistro rosdep key?

Reply:
> do I also need to change to ifcfg_vendor on rosdistro rosdep key?

I don't think so. `ifcfg-pip` is the right name for the rosdistro key (the key isn't needed for this PR since there's a vendor package). The package.xml should be changed to depend on the new package `ifcfg_vendor`.